lib/openshift/node.rb

```diff
@@ -73,6 +73,12 @@ def schedulable?(user: nil, cached: true, quiet: false)
       return !spec['unschedulable']
     end
 
+    def is_worker?(user: nil, cached: true, quiet: false)
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      current_config = rr.dig('metadata', 'annotations', 'machineconfiguration.openshift.io/currentConfig')
+      current_config.start_with? 'rendered-worker'
```

Comment:
is this role documented? I wonder how reliable such check is.

Reply:
yeah, I debated about that...an alternative is to check for the existence of the label `node-role.kubernetes.io/master=` for a node that's a `master` or  `node-role.kubernetes.io/worker=`.  That's probably more reliable.

Reply:
+1 to use `node-role`